.github/workflows/push-binary-dispatch.yml

```diff
@@ -46,7 +43,7 @@ jobs:
       - run: yarn
       - run: yarn install --frozen-lockfile
       - run: yarn build
-      - run: yarn test
+      - run: yarn test # Or shall the test step be skipped?
```

Comment:
@tplooker 
If this `build_win_pub_binaries` job is required, maybe have to skip the test step?